Installation/Schema/klant.klantcontact.schema.json

```diff
@@ -2,16 +2,16 @@
   "required": [
     "geregistreerdOp",
     "indicatieContactMislukt",
     "nummer",
     "onderwerp",
-    "plaatsgevondenOp",
-    "actors"
```

Comment:
necessary because there are contactmomenten posted by the frontend without attendees/actors